module-services/service-desktop/endpoints/developerMode/DeveloperModeHelper.cpp

```diff
@@ -66,6 +68,34 @@ namespace sdesktop::endpoints
             code       = toCode(owner->bus.sendUnicast(msg, ServiceCellular::serviceName));
             return {sent::delayed, std::nullopt};
         }
+        else if (body[json::developerMode::EQ].is_string()) {
```

Comment:
It would be nice to add new types of requests to https://github.com/mudita/QAMuditaHarness/blob/master/api/developermode.py instead of writing it directly here.

Reply:
added in PR:
https://github.com/mudita/QAMuditaHarness/pull/44